roles/setup_gitea/tasks/install.yml

```diff
@@ -0,0 +1,337 @@
+---
+- name: Create namespace for Gitea
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ sg_namespace }}"
+
+- name: Create Service Account
+  community.kubernetes.k8s:
+    definition:
+      kind: ServiceAccount
+      apiVersion: v1
+      metadata:
+        name: anyuid
+        namespace: "{{ sg_namespace }}"
+
+- name: Create Role
+  community.kubernetes.k8s:
+    definition:
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: Role
+      metadata:
+        name: scc-anyuid
+        namespace: "{{ sg_namespace }}"
+      rules:
+        - apiGroups:
+            - security.openshift.io
+          resourceNames:
+            - anyuid
+          resources:
+            - securitycontextconstraints
+          verbs:
+            - use
+
+- name: Create RoleBinding
+  community.kubernetes.k8s:
+    definition:
+      kind: RoleBinding
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: sa-to-scc-anyuid
+        namespace: "{{ sg_namespace }}"
+      subjects:
+        - kind: ServiceAccount
+          name: anyuid
+      roleRef:
+        kind: Role
+        name: scc-anyuid
+        apiGroup: rbac.authorization.k8s.io
+
+- name: Generate JWT secret
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret JWT_SECRET
+  changed_when: false
+  register: _sg_jwt_secret
+
+- name: Generate the LFS JWT secret
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret LFS_JWT_SECRET
+  changed_when: false
+  register: _sg_lfs_jwt_secret
+
+- name: Generate the internal token
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret INTERNAL_TOKEN
+  changed_when: false
+  register: _sg_internal_token
+
+- name: Generate the secret key
+  ansible.builtin.command: /usr/bin/podman run -ti {{ sg_gitea_image }} /usr/local/bin/gitea generate secret SECRET_KEY
+  changed_when: false
+  register: _sg_secret_key
+
+- name: Create Config Map with the Gitea configuration
```

Comment:
couldn't we use separate template files for this kind of resources that are really long? same for others that come next

Reply:
done